### DISCUSSION TITLE
LDAP Configuration for Wiki.js

### DISCUSSION THREAD

**Discussion by agnelbruce** [requarks/wiki#3376] [Help / Questions]
**Question**
Kindly help me to setup LDAP in wiki.js.

**Host Info (please complete the following information):**
OS: UBUNTU SERVER 18.04
Wiki.js version: 2.3.81
Database engine:MSSQL

**Comment by NGPixel** [comment]
Can you be more specific? What exactly do you need help with?

**Comment by agnelbruce** [comment]
Hai,

    I am testing between windows server 2012 and wiki js using LDAP AD,dont know to fill all the fields correctly.
![github query](https://user-images.githubusercontent.com/65269106/82543009-9237e300-9b70-11ea-83db-390ec2aaa8d5.JPG)

**Comment by agnelbruce** [comment]
can any one helpme to setup the ldap authentication pls.

**Comment by winkelement** [comment]
This is the schema i use to connect to an Apple Open Directory so i guess it is not so helpful. I guess you have to adjust your admin bind dn. Maybe research on how to test your settings from the command line to figure out what is wrong.

![Bildschirmfoto 2020-06-03 um 17 53 19](https://user-images.githubusercontent.com/3525191/83664174-ee1b5680-a5c9-11ea-8ab9-2229efe83414.png)
